### PR TITLE
Detail card

### PR DESCRIPTION
**Pull request by DanCesca**
Hello! 

With this pull request I have:

- added Button component
- modified and amplified TasteCard component to show more details when selected
- added Sample coffees with image to the hard coded database, so that I can show a sample coffee when the card is selected
- added a toggle function to the button, so that it opens and closes the card, hereby modifying its appearance 

Thank you very much in advance for your feedback and review!

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function App() {
   return (
     <AppContainer>
       <h1>Coffee Roast Levels</h1>
-      <CardWrapper>
+      <CardWrapper role="list">
```
Comment (julialoeschel) [inline client/src/App.js:9]: very nice!

File: client/src/database.js [hunk 2]
```diff
@@ -3,23 +3,56 @@ export const tastes = [
     id: '1',
     name: 'Light Roast',
     image: require('./images/light-roast.jpg'),
+    teaser: 'You will like this, if you enjoy fruity flavours and a striking acidity.',
     profile:
       'Coffees with this roast present light and fresh flavours. Nuances of berries, fruits and citrus. The taste varies from citrusy to fruity and sweet.',
+    sample: {
+      id: '1',
+      name: 'El Amanecer',
+      image: require('./images/sample-light.jpg'),
```
Comment (julialoeschel) [inline client/src/database.js:12]: Why would you use the file-loader here? Is it really necessary ? If yes, let me know why 🦆 🙂

**Review by julialoeschel — COMMENTED** [review]
great job there! :octocat: 
I have only one question about the file loader.

Other than that is there no deployment, it would be great to see your work very soon on Vercel/heroku.

File: client/src/database.js (hunk 2 above)
Comment (DanCesca) [reply client/src/database.js:12]: Hey! Thank you very much for your feedback. Well, the simple linking did not work, so I asked google and found this solution. ;) But if you have a suggestion for another workaround I'm more than happy to change that!

**Pull request merged**